### ISSUE TITLE
Can Gimbal  be transplanted to other models？

### ISSUE THREAD

**Issue by lixiaoyangyangyang** [PX4/PX4-Autopilot#20748]
At present, only the typhoon_h480 model is adapted to the Gimbal in the simulation. Can this Gimbal be transplanted to other simulation models? thank you.

**Comment by lixiaoyangyangyang** [comment]
Thank you for your reply，but how much should this parameter be set？I want to add the gimbal to standard_vtol now. After transplanting the relevant code in typhoon_h480, I can only see the camera shaking in gazebo and can't rotate.

**Comment by df8s7d9g8s** [comment]
In the airframe file (init.d-posix/airframes/1040_standard_vtol) set MNT_MODE_IN to 4 and MNT_MODE_OUT to 2, you can look at the airframe for Typhoon in the same folder for reference. Also make sure to set up the mavlink_interface and gimbal plugins in the model file.

**Comment by lixiaoyangyangyang** [comment]
After I set MNT_MODE_IN to 4, the qgc displays as shown in the [figure.]![2](https://user-images.githubusercontent.com/100847998/207228849-f53cfa2d-5c75-4a31-8a06-17e3d2d36fad.png)
Is my px4 version too low? I will Mavlink_interface and gimbal plug are placed in standard_ Vtol SDF, but still cannot rotate. In qgc, if MAV_TYPE is changed to 13, the gimbal can rotate, but  MAV_TYPE corresponding to standard_vtol is 22
![1](https://user-images.githubusercontent.com/100847998/207229099-937e4e9a-47be-4404-a740-87912849e618.png)
![3](https://user-images.githubusercontent.com/100847998/207229112-dfd9f5d3-d759-4d6d-bac4-d38b04f4c224.png)

**Comment by df8s7d9g8s** [comment]
Yeah looks like Gimbal protocol V2 wasn't introduced until PX4 1.12, but in an older version you could use whatever the Typhoon uses. The MAV_TYPE thing sounds like a bug I guess.

**Comment by lixiaoyangyangyang** [comment]
Thank you for your reply. I will try to use the new version of PX4 later, but I am just starting to use PX4. There is less information about this in China. If possible, I hope you can provide me with a contact information. Thank you very much.
